astropy/coordinates/tests/test_angular_separation.py

```diff
@@ -46,58 +45,15 @@ def test_angsep():
             assert np.fabs(angsep - conv(corrsep)) < conv(correctness_margin)
 
 
-def test_fk5_seps():
-    """
-    This tests if `separation` works for FK5 objects.
-
-    This is a regression test for github issue #891
-    """
-    a = FK5(1.0 * u.deg, 1.0 * u.deg)
-    b = FK5(2.0 * u.deg, 2.0 * u.deg)
-    a.separation(b)
-
-
 def test_proj_separations():
```

Comment:
Change the name? Maybe `test_cannot_add_subtract()`?

Reply:
Indeed, you are right that the test should be moved... While I agree the patch is large, I already reviewed it, and am happy to just have a separate commit that moves this test. But also OK to have a separate PR...